upgrading/manual_upgrades.adoc

```diff
@@ -890,8 +890,8 @@ provides the example JSON files:
 ----
 endif::[]
 ifdef::openshift-enterprise[]
-By default, the xref:../../install_config/install/quick_install.adoc#install-config-install-quick-install[quick] and
-xref:../../install_config/install/advanced_install.adoc#install-config-install-advanced-install[advanced installation]
+By default, the xref:../install_config/install/quick_install.adoc#install-config-install-quick-install[quick] and
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[advanced installation]
```

Comment:
In some of the code blocks below, we can probably drop the ==== around the ---- format

Reply:
Fixt